src/reactive-handler.ts

```diff
@@ -178,11 +182,16 @@ export class ReactiveProxyHandler {
         // if the descriptor has a value, as opposed to getter/setter
         // So we can just check if writable is present and then see if
         // value is present. This eliminates getter and setter descriptors
-        if (hasOwnProperty.call(descriptor, 'writable') && !hasOwnProperty.call(descriptor, 'value')) {
-            const originalDescriptor = getOwnPropertyDescriptor(originalTarget, key) as PropertyDescriptor;
+        const originalDescriptor = getOwnPropertyDescriptor(originalTarget, key);
```

Comment:
this change is needed to accommodate for the new fact that sometimes the key is not installed on the original target, but on the shadow for the magic key